.github/workflows/vulnerability-scans.yml

```diff
@@ -57,6 +57,16 @@ jobs:
           ignore-unfixed: true
           vuln-type: os
           scanners: vuln,secret
+          # Enables caching of trivy config file that is used to search for the
+          # vulnerabilities. We cache it since it only updates once every 6 hours,
+          # and the confg hosts can be down, causing failures
+          cache: 'true'
+        env:
+          # Set to skip since we cache the files in the vulnerability-update-cache
+          # workflow
+          TRIVY_SKIP_DB_UPDATE: true
+          TRIVY_SKIP_JAVA_DB_UPDATE: true
```

Comment:
Looks like this will fail if the cache is missing. This could block PRs or any updates to child repos until the cache workflow is merged and ran